### PR TITLE
[DOC,FIX] Check links

### PR DESCRIPTION
**Pull request by eseiler**
See also https://github.com/seqan/www.seqan.de/pull/59

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ assignees: ''
   <details><summary>Click for detailed steps</summary><br>
 
   Check for [new releases](https://github.com/PeterFeicht/cppreference-doc/releases) and update the link and hash
-  in the [code base](https://github.com/seqan/seqan3/blob/b0b279689fa65c2431a5162f2d8acc3ca663f72d/test/documentation/seqan3-doxygen.cmake#L37).
+  in the [code base](https://github.com/seqan/seqan3/blob/77c5e8a1381ed39ae96a6966c008f56dd15152cb/test/documentation/seqan3-doxygen.cmake#L54).
```
Comment (eseiler) [inline .github/ISSUE_TEMPLATE/release-checklist.md:21]: Updated link

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -105,8 +105,8 @@ assignees: ''
 ---
 
 - [ ] Merge release branch into master.
-- [ ] Bump succeeding version number in master. ([include/seqan3/version.hpp](https://github.com/seqan/seqan3/blob/3.0.2/include/seqan3/version.hpp#L19-L24))
-- [ ] Bump latest stable version number of the API-Stability test in master. ([test/api_stability/CMakeLists.txt](https://github.com/seqan/seqan3/blob/3.0.3/test/api_stability/CMakeLists.txt#L10))
+- [ ] Bump succeeding version number in master. ([include/seqan3/version.hpp](https://github.com/seqan/seqan3/blob/77c5e8a1381ed39ae96a6966c008f56dd15152cb/include/seqan3/version.hpp#L18-L23))
```
Comment (eseiler) [inline .github/ISSUE_TEMPLATE/release-checklist.md:108]: Updated link

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -105,8 +105,8 @@ assignees: ''
...
+- [ ] Bump latest stable version number of the API-Stability test in master. ([test/api_stability/CMakeLists.txt](https://github.com/seqan/seqan3/blob/77c5e8a1381ed39ae96a6966c008f56dd15152cb/test/api_stability/CMakeLists.txt#L10))
```
Comment (eseiler) [inline .github/ISSUE_TEMPLATE/release-checklist.md:109]: Fixed link

File: CHANGELOG.md [hunk 4]
```diff
@@ -114,7 +114,7 @@ applicable, when it will be removed. We recommend upgrading version-by-version t
 You can either directly check the reported code or verify with our documentation how the new API should be used.
 
 For a complete list of behavioural changes in our public and internal API, you can consult our API stability
-regression test suite and patches at https://github.com/seqan/seqan3/tree/master/test/api_stablility/3.0.2/.
+regression test suite and patches at https://github.com/seqan/seqan3/tree/master/test/api_stability/3.0.2.
```
Comment (eseiler) [inline CHANGELOG.md:117]: Fixed link

**Review by SGSSGene — APPROVED** [review]
:+1:

**Review by marehr — APPROVED** [review]

**Pull request merged**